pkgs/development/compilers/emscripten/bintools.nix

```diff
@@ -0,0 +1,66 @@
+{ emscripten
+, stdenvNoCC, lib, makeWrapper
+, wasmArch ?
+  if stdenvNoCC.targetPlatform.isWasm
+  then stdenvNoCC.targetPlatform.uname.processor
+  else "wasm32"
+, wrapBintoolsWith
+}:
+
+let
+  stdenv = stdenvNoCC;
+  targetPrefix = "${wasmArch}-emscripten-";
+  llvm = emscripten.llvmEnv;
+
+unwrapped = stdenv.mkDerivation {
```

Comment:
wrong indentation,  run `git diff --name-only HEAD~ --diff-filter=A | grep "\.nix" | xargs nixpkgs-fmt` to format added files only